### ISSUE TITLE
hub.docker.com reports "build failure", when all is fine

### ISSUE THREAD

**Issue by jnweiger** [owncloud-docker/server#170]
The entry page https://hub.docker.com/r/owncloud/server/
links to https://drone.owncloud.com/owncloud-docker/server where there is nothing.

Our images are built at https://cloud.drone.io/owncloud-docker/server

**Comment by tboerger** [comment]
Feel free to add a pipeline similar to this to get the DockerHub in sync with the README.md:

```
def readme(ctx):
  return {
    'kind': 'pipeline',
    'type': 'docker',
    'name': 'readme',
    'platform': {
      'os': 'linux',
      'arch': 'amd64',
    },
    'steps': [
      {
        'name': 'execute',
        'image': 'sheogorath/readme-to-dockerhub:latest',
        'pull': 'always',
        'environment': {
          'DOCKERHUB_USERNAME': {
            'from_secret': 'docker_username',
          },
          'DOCKERHUB_PASSWORD': {
            'from_secret': 'docker_password',
          },
          'DOCKERHUB_REPO_PREFIX': 'owncloud',
          'DOCKERHUB_REPO_NAME': 'server',
          'SHORT_DESCRIPTION': 'ownCloud community edition server stack ',
          'README_PATH': 'README.md',
        },
      },
    ],
    'depends_on': [
      'changelog',
    ],
    'trigger': {
      'ref': [
        'refs/heads/master',
      ],
    },
  }
```

**Comment by wkloucek** [comment]
fixed by #226
